### ISSUE TITLE
Memote Git error

### ISSUE THREAD

**Issue by parsanayyara** [opencobra/memote#763]
Hello, 

I am trying to generate a memote report over a local git commit of a model and ran across this issue when trying to run 

memote report history --location <my storage folder>.

![image](https://github.com/opencobra/memote/assets/77234398/3b562be8-457a-486e-a7b1-8b5b129ca4e4)

Followed the flowchart on https://memote.readthedocs.io/en/latest/flowchart.html but the error keeps showing up everytime I ran memote report. Could anyone help point out what the issue might be?

**Comment by carrascomj** [comment]
It seems like your git repository does not contain the deployment branch "gh-pages". You can create it yourself with:

```
git checkout -b gh-pages
```

Did you start the repository with `memote new` or manually (`git init`)?
* If you started it with `memote new` and still have this issue, could you paste your `memote.ini` here? There might be a bug with properly using the deployment branch.
* If you started it with `git init` you may need other preparations for the repository that `memote new` performs (like generating a `memote.ini` and setting up the branches.)

Of course, if you only need a snapshot report, try:

```bash
memote report snapshot /path/to/model
```

**Comment by parsanayyara** [comment]
Hello, I resolved the previous error, but now am getting a new one related to the cookiecutter. 
![image](https://github.com/opencobra/memote/assets/77234398/404cf44c-0fca-4a10-9d44-230cb54e96d3)
![image](https://github.com/opencobra/memote/assets/77234398/2965997c-e36b-4204-8368-7be8177126ec)

It looks to me like it could not find the 'master' branch; might be because the recent Git update that changes master to main?

**Comment by carrascomj** [comment]
Yes, that seems to be the problem, that must be fixed for it to work.

**Comment by parsanayyara** [comment]
Is there any possible way around it that doesn't require modifying the code? Would it work if I rename main to master maybe?

**Comment by Midnighter** [comment]
> Is there any possible way around it that doesn't require modifying the code? Would it work if I rename main to master maybe?

We should change the cookiecutter to accept both master and main branches. Renaming the branch at this point won't help. You would have to manually perform the missing steps from the post generation hook.

**Comment by parsanayyara** [comment]
Has there been any updates on this bug? Can both master and main be accepted by the cookiecutter?
